posthog/temporal/batch_exports/s3_batch_export.py

```diff
@@ -360,6 +361,25 @@ async def initialize_and_resume_multipart_upload(inputs: S3InsertInputs) -> tupl
     return s3_upload, interval_start
 
 
+def s3_default_fields() -> list[BatchExportField]:
+    """Default fields for an S3 batch export.
+
+    Starting from the common defualt fields, we add and tweak some fields for
```

Comment:
nit: `default`

(I never know why Github sometimes does and doesn't show me the diff recommender thing.)